CHANGELOG.md

```diff
@@ -29,6 +29,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   Dashboard, documenting all endpoints and specific data fields used by the
   frontend ([#1566](https://github.com/o1-labs/mina-rust/issues/1566))
 
+- **FEATURE**: Add logging if initial peers are invalid
```

Comment:
```suggestion
- **Feature**: Add logging if an initial peer is invalid
```

FEATURE -> Feature: there is no need to capitalize the word.
Plural -> singular: one initial peer is enough to have a log.